tutorials/01-Command-Line/Challenge 1/Text Files/generate.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+for i in `seq 1 4500`; do
+  if (($i <= 600)) 
+  then
+            random=$((1 + RANDOM % 39))
+            if (( $random % 2 ))
+            then
+               sed -n $((1 + RANDOM % 39))p ./More\ Text.txt > "xaaaa$(echo $i | tr '[0-9]' '[a-j]').txt" 
```

Comment:
```suggestion
               sed -n "$((1 + RANDOM % 39))p" "./More Text.txt" > "xaaaa$(echo $i | tr '[0-9]' '[a-j]').txt" 
```

Reply:
Resolved in https://github.com/madetech/learn/pull/165/commits/e0ff53f07eb4edbfee6d84c04bad27521816de7e